views/register.hbs

```diff
@@ -29,6 +29,18 @@
             <input id="email" name="email" type="text" data-auto-trim placeholder="{{{__ "enter email"}}}" value="{{formValues.email}}" size="30" class="pure-input-1 login-form-input">
         </div>
 
+        {{#if signupTeamsTransfer}}
+        <div class="pure-control-group">
+          <h4>Teams suggested by invite link, check box to join</h4>
```

Comment:
May want to add explanation of what teams are or link to explanation of teams

Reply:
We'll also need to internationalize the text as we get closer to merge